app/assets/javascripts/reducers/columns/backlog.js

```diff
@@ -4,14 +4,45 @@ const initialState = {
   stories: [],
 };
 
+const filterByState = state => story => {
+  return story.state === state;
+}
+
+
+const orderByState = (stories) => {
+  const ordered = [...stories];
+
+  ordered.sort((a, b) => {
+    return parseFloat(a.position) - parseFloat(b.position);
```

Comment:
This function seems to be the same as this one https://github.com/Codeminer42/cm42-central/pull/356/files#diff-859db19caeda89d65deecbdbc08bd544R12. Even better we could extract it to a separate file and having something like `ordered.sort(Story.comparePositions)`, this `Story` object could be inside a file in this folder https://github.com/Codeminer42/cm42-central/tree/master/app/assets/javascripts/models/beta. What do you think?